### ISSUE TITLE
cp: can't create '@***:~/': No such file or directory

### ISSUE THREAD

**Issue by kojofosu-ghyrate** [marcodallasanta/ssh-scp-deploy#13]
why do i keep getting this error please. any help?

**Comment by marcodallasanta** [comment]
Hi @kojofosu-ghyrate , could you please add your configuration to reproduce this?

**Comment by kojofosu-ghyrate** [comment]
```yml
      - name: Deploy snapshot
        uses: marcodallasanta/ssh-scp-deploy@v1.0.4
        with:
          local: './target/ServerLookup-1.0-SNAPSHOT.jar'                                                  # Local file path - REQUIRED false - DEFAULT ./
          remote: '~/'                                                 # Remote file path - REQUIRED false - DEFAULT ~/
          host: ${{secrets.HOST}}                                      # Remote server address - REQUIRED true
          port: ${{secrets.PORT}}                                      # Remote server port - REQUIRED false - DEFAULT 22
          user: ${{secrets.USER}}                                      # Remote server user - REQUIRED true
          password: ${{secrets.PASSWORD}}     
```

**Comment by marcodallasanta** [comment]
@kojofosu-ghyrate could you please provide the OS and version of your host? Furthermore does your user have an home directory on your host?  

If you login into your host with the user you configured in the action, what's the output of this?

```bash
cd ~ && pwd
```

**Comment by kojofosu-ghyrate** [comment]
@marcodallasanta cd ~ && pwd outputs `"/home/botservice/"`

**Comment by kojofosu-ghyrate** [comment]
<img width="678" alt="deploy1 0 5" src="https://user-images.githubusercontent.com/69309530/132663547-fd0522b1-4d93-45a2-9c28-a4e8faf1ff62.PNG">

```
cp: can't create '@***:~/': No such file or directory
/with_pass.sh: line 17: exit: 1 : numeric argument required
````

**Comment by marcodallasanta** [comment]
### First part:

```bash
cp: can't create '@***:~/': No such file or directory
```

I think is a problem related to your configuration and not a bug, I suggest you maybe to change  `local` from a relative to an absolute path without using `~` , maybe like this you can figure out something like if this is a permission or missing directory problem or what so ever. Hope this can help you: https://stackoverflow.com/questions/39847496/cp-cannot-create-directory-no-such-file-or-directory .

I'm considering if this error pops out only for your distro, which distro are you using @kojofosu-ghyrate ?

## Second part :

```bash
/with_pass.sh: line 17: exit: 1 : numeric argument required
```

I think that yesterday during the fix of #12 I messed up something. But this is less important because now the action fails and this is what we was searching for 🔥 🔥 🔥 
So We can ignore this safely till I'll fix this when I'll have enough time.

**Comment by kojofosu-ghyrate** [comment]
@marcodallasanta  Sorry for delay in response, we're setting up a test environment to try . with this action. If any issues come up again, i'll kindly alert you
